python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -46,7 +46,7 @@ def create_while_node(condition_name, body_name, loop_var_names):
     # For example: loop_var_names = [a, b, foo.x], the type of `a` or `b` is gast.Name,
     # but the type of `foo.x` gast.Attribute.
 
-    while_func_name = "fluid.dygraph.dygraph_to_static.convert_operators.convert_while_loop"
+    while_func_name = "paddle.jit.dygraph_to_static.convert_operators.convert_while_loop"
```

Comment:
Same as above

Reply:
I think it is better just be same as that in the past. Even if we would like to change, it is better to be in another PR.